src/docs/firmware/flashing-firmware.md

```diff
@@ -14,13 +14,17 @@ Refer to [this page](about) for an overview of both RW_LEGACY and UEFI.
 To convert your machine from Google's firmware and ChromeOS to Coreboot and AltOS (Linux/Windows/macOS), you will need to first perform the following steps:
 
 1. [Enable developer mode](developer-mode).
-2. [Disable write protect](write-protect).
+2. [Disable write protect](write-protect) (If flashing UEFI).
 3. Run [MrChromebox's firmware utility script](https://mrchromebox.tech/#fwscript) within VT-2 (`ctrl` + `alt` + `f2` (right arrow)).
     * Type `cd; curl -LO mrchromebox.tech/firmware-util.sh && sudo bash firmware-util.sh` and press Enter.
     * If you encounter certificate related errors when downloading the script from ChromeOS, then add `-k` to the `curl` and script command to bypass SSL certificate checking as so:
         * `cd; curl -LOk mrchromebox.tech/firmware-util.sh && sudo bash firmware-util.sh`
 4. If you are flashing Full ROM, the firmware script will prompt you to make a backup of your stock firmware. **Store it in a safe place** (Google Drive, another PC, etc), as reverting to stock firmware without a backup is **very difficult**.
-5. Shutdown the device.
```

Comment:
I think we should still have a step 5 here, maybe something like `5. If there is no error, shutdown the device without worry`. Repetitiveness of something important isn't bad (to an extent)